doc/mesh-api.txt

```diff
@@ -1061,6 +1079,68 @@ Properties:
 		Uniform Resource Identifier points to out-of-band (OOB)
 		information (e.g., a public key)
 
-Mesh Node Configuration Schema
-==============================
-<TBD>
+Mesh Node Configuration Examples
+================================
+Example of Json format for ImportLocalNode():
+
+	Import local node operation:
```

Comment:
Keep it consistent: "Simplified" and "Full" representation, docs for the API don't mention any "migration"